### ISSUE TITLE
Can't get inner text of a script

### ISSUE THREAD

**Issue by LiteracyFanatic** [fsprojects/FSharp.Data#1292]
Is there some reason getting the inner text of a script element is not allowed? I'm working on a webscraping project where I need need to obtain a cookie by evaluating some javascript from one of the pages. I'm shelling out to Node.js to do that, but the rest of the HTML parsing and logic is all in F#.  I can work around this by manually stripping the script tags myself, but it'd be much more convenient to just call `HtmlNode.innerText`. It looks like this restriction was introduced in commit a715fbc8ac1b40ebe5b4aa9bb1bbc695e38d4b6b, but I can't see any explanation as to why.

**Comment by pedersenjens** [comment]
Yes, why is it that that inner text is not available for script tags and apparently also not style tags? Is there a good reason for this change?

**Comment by tnishimura** [comment]
I also find this an odd bug, any chance of reverting the changes?

**Comment by dsyme** [comment]
I don't know of any intrinsic reason - we can't revert a change from that long ago but please do submit a PR to address this if you can
